dev/run-tests-jenkins

```diff
@@ -26,4 +26,11 @@ FWDIR="$( cd "$( dirname "$0" )/.." && pwd )"
 cd "$FWDIR"
 
 export PATH=/home/anaconda/bin:$PATH
```

Comment:
actually, we set this in the jenkins worker config, spark PRB build config, and here...  we can safely remove this line.